test/Microsoft.AspNet.FileProviders.Combined.Tests/TestUtility/MockChangeToken.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.Extensions.Primitives;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.FileProviders.Combined.Tests.TestUtility
```

Comment:
As for the name, I have put it in this folder to be consistent with the FileSystemGlobbing test project.
Do I move the three files ?